src/app/DTT/address/get_address/helpers/set-address-suggestions.js

```diff
@@ -0,0 +1,8 @@
+import { equals } from 'ramda'
+const setAddressSuggestions = (searchAddress, allOptions, setAllOptions) => {
+  const displaySuggestions = predictions =>
+    !equals(allOptions, predictions) && setAllOptions(predictions)
+  const service = new window.google.maps.places.AutocompleteService()
```

Comment:
why is this service being declared inside the function. you will get a new service every single time